client/stylesheets/_toCat.scss

```diff
@@ -2311,6 +2312,12 @@ i:hover {
     @include headerUsername;
 } */
 
+.headerNavMax {
+	margin: 0 auto;
+	width: 100%;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered margin, max-width, width